business_requirement_deliverable_project/models/project.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.tools.translate import _
+from openerp.osv import osv
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    origin = fields.Char('Source Document')
+
+    @api.multi
+    def generate_project_wizard(self):
+        br_ids = self.env.context.get('br_ids', False)
+        from_project = False
+        if not br_ids:
+            br_ids = self.br_ids
+            from_project = True
+        default_uom = self.env['project.config.settings'].\
+            get_default_time_unit('time_unit')
+        default_uom = default_uom.get('time_unit', False)
+        if not default_uom:
+            raise osv.except_osv(
```

Comment:
@moylop260 I would rather keep it since the others modules has passed with this deprecated feature and don't have impact with Travis, Runbot but change for the next version.

@elicoidal @dreispt @seb-elico ?